.github/workflows/build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           file: ./coverage/coverage.txt
       -
         name: Expose GitHub Runtime
-        uses: crazy-max/ghaction-github-runtime@v1
+        uses: crazy-max/ghaction-github-runtime@94085cef04e835448ac610090128515e781b4939
```

Comment:
Why is this changed from a tag -> commit hash? It looks like the same commit https://github.com/crazy-max/ghaction-github-runtime/releases/tag/v1 

(and also a couple of other places below)

Reply:
Yes as I said:

> also pin external/untrusted actions being used (dependabot still track sha linked to semver releases so it's still usable this way).

Because Git tags are mutable we don't want to take the risk to fetch unchecked changes.